binding/Binding/SKDrawable.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+using NativePointerDictionary = System.Collections.Concurrent.ConcurrentDictionary<System.IntPtr, SkiaSharp.SKDrawable>;
+
+namespace SkiaSharp
+{
+	public class SKDrawable : SKObject
+	{
+		private static readonly NativePointerDictionary managedDrawables = new NativePointerDictionary ();
+
+		// delegate declarations
+		[UnmanagedFunctionPointer (CallingConvention.Cdecl)]
+		internal delegate void draw_delegate (IntPtr managedDrawablePtr, IntPtr canvasPtr);
+		[UnmanagedFunctionPointer (CallingConvention.Cdecl)]
+		internal delegate void getBounds_delegate (IntPtr managedDrawablePtr, out SKRect rect);
+		[UnmanagedFunctionPointer (CallingConvention.Cdecl)]
+		internal delegate SKPicture newPictureSnapshot_delegate (IntPtr managedDrawablePtr);
```

Comment:
I believe this has to be `IntPtr` / `sk_picture_t` because this is an interop function. The equivalent is to pass the handle around:

```suggestion
		internal delegate IntPtr newPictureSnapshot_delegate (IntPtr managedDrawablePtr);
```